benchmarks/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ruby:3.3.0-bullseye
+
+# Make apt non-interactive
+RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci \
+  && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90circleci
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install required packages
+RUN set -ex; \
+        apt-get update; \
+        mkdir -p /usr/share/man/man1; \
+        apt-get install -y --no-install-recommends \
+            git mercurial xvfb \
+            locales sudo openssh-client ca-certificates tar gzip parallel \
+            net-tools netcat unzip zip bzip2 gnupg curl wget \
+            tzdata rsync vim less jq; \
+        rm -rf /var/lib/apt/lists/*;
```

Comment:
Out of curiosity, why does this use `;` and not `&&` between commands?

Reply:
No reason, just copy/paste from existing solution.
I changed it to `&&` as suggested.